### ISSUE TITLE
False positives when there are several platforms on the Gemfile

### ISSUE THREAD

**Issue by calavera** [jruby/jruby-lint#4]
My Gemfile contains this:

```
platforms :ruby do
  gem 'sqlite3'
end
platforms :jruby do
  # SQLite JDBC adapter
  gem 'jdbc-sqlite3', :require => false
end
```

but I get the warning about the native sqlite3 gem:

```
./Gemfile:9: [gems, warning] Found gem 'sqlite3' which is reported to have some issues: Use activerecord-jdbc-adapter instead along with jdbc-sqlite3.
```
